src/drt/README.md

```diff
@@ -1,4 +1,4 @@
-# TritonRoute
+# Detailed Routing
 
 TritonRoute is an open-source detailed router for modern industrial
```

Comment:
The Detailed Routing (`drt`) module in OpenROAD is based on the open-source Triton Route...

Reply:
Replicate similar phrase for other tools.